Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -426,11 +428,16 @@ - (void)popped
     [self hideInputIfNeeded];
 }
 
-- (void)configureForThread:(TSThread *)thread action:(ConversationViewAction)action
+- (void)configureForThread:(TSThread *)thread
+                    action:(ConversationViewAction)action
+            focusMessageId:(nullable NSString *)focusMessageId
```

Comment:
If there's a name you prefer over `focusMessageId`, I'm open to it.

Reply:
Not bad!